tests/new_functional/config.js

```diff
@@ -44,48 +44,47 @@ let config = {
   }
 };
 
-const sauceLabsConfig = {
-  services: ['sauce'],
-  sauceConnect: true,
-  user: process.env.SAUCE_USERNAME,
-  key: process.env.SAUCE_ACCESS_KEY,
-  capabilities: [capabilities.firefox]
-};
-
 const browserStackConfig = {
   services: ['browserstack'],
   user: process.env.BROWSERSTACK_USER,
   key: process.env.BROWSERSTACK_ACCESS_KEY,
   browserstackLocal: true
 }
 
-const phantomjsConfig = {
-  services: ['phantomjs'],
-  waitforTimeout: 3000,
-  capabilities: [capabilities.phantomjs],
-  maxInstances: 4,
-  phantomjsOpts: {
-    ignoreSslErrors: true
+const phantomjsConfig = Object.assign(
+  {},
+  config,
+  {
+    services: ['phantomjs'],
+    waitforTimeout: 3000,
+    capabilities: [capabilities.phantomjs],
+    maxInstances: 4,
+    phantomjsOpts: {
+      ignoreSslErrors: true
+    },
+    before: function() {
+      chaiAsPromised.transferPromiseness = browser.transferPromiseness;
+      browser.setViewportSize({
+        width: 1280,
+        height: 1024
+      });
+    }
   }
-
-};
+);
```

Comment:
What was the reason for having to extend the Phantom config. Was this required to get the chrome headless working or is it an unrelated change? Sorry just being nosey on a Runner PR since it would be nice to get the headless chrome running on Author as well :-)

Reply:
Unrelated. I almost changed back to running Phantom to unblock us and realised that I'd broken the configuration in the move to Chrome headless. So this is really re-instating some of that config so we can easily switch back to Phantom if we need to.